repos/fdbt-site/fdbt-types/matchingJsonTypes.ts

```diff
@@ -273,6 +270,7 @@ export interface TicketPeriod {
 }
 
 export interface BaseProduct {
```

Comment:
I think we need to put these types back to how they were, because productName is only *always* inside the product after the ticket has gone through the exporter - in the site, it might not have a product name. Sorry for misleading you on this!

Reply:
[1eb22e6](https://github.com/fares-data-build-tool/create-data/pull/578/commits/1eb22e63ffd9b58ee969e33b1648e09d7e69f501)